src/V3Dfg.h

```diff
@@ -280,6 +149,10 @@ class DfgVertex VL_NOT_FINAL {
 public:
     virtual ~DfgVertex() VL_MT_DISABLED;
 
+    // List type that can store Vertex (which must be a DfgVertex) instances via m_links
+    template <typename Vertex>
+    using List = V3List2<DfgVertex, &DfgVertex::m_links, Vertex>;
```

Comment:
And they you create a typedef which is the list type that holds elements via the given member. (you can have different types that use different members to store an element in multiple lists, like V3GraphEdge does.